### DISCUSSION TITLE
Getting the stored procedures and functions defined in a given DB schema

### DISCUSSION THREAD

**Discussion by sr-murthy** [sqlalchemy/sqlalchemy#6551] [Usage Questions]
I don't see a way currently to find out what stored procedures and functions are defined in a given DB schema. I've looked at what's available via the DB inspection tools  (https://docs.sqlalchemy.org/en/14/core/reflection.html#sqlalchemy.engine.reflection.Inspector) but it doesn't seem to have a way of listing procedures and functions.

**Comment by zzzeek** [comment]
SQLAlchemy does not have this capability included, you will have to query the information schema views of your database manually.

**Reply by sr-murthy** [reply]
OK, thanks.
